tools/process_headers.zig

```diff
@@ -354,7 +354,8 @@ pub fn main() !void {
                         .Directory => try dir_stack.append(full_path),
                         .File => {
                             const rel_path = try std.fs.path.relative(allocator, target_include_dir, full_path);
-                            const raw_bytes = try std.io.readFileAlloc(allocator, full_path);
+                            const max_size = 4 * 1024 * 1024;
```

Comment:
Any particular reason to use this max size instead of `std.math.maxInt(usize)`?

Reply:
No, `4 * 1024 * 1024` was pulled out of thin air. Would `std.max.maxInt(usize)` be better? I had the impression that that isn't good style as many other reads in other tools are limited, for example:

https://github.com/ziglang/zig/blob/fd47839064775c0cc956f12a012f0893e1f3a440/src-self-hosted/main.zig#L15

Perhaps a larger limit would be more appropriate.

Reply:
Yes maybe maxInt is not a good idea, 4 MB seems pretty low though.

Reply:
What limit would be more appropriate?

Reply:
I guess 2 GiB sounds good, although maybe 4 MiB is enough for the headers that are preprocessed so feel free to ignore me as well, I was just wondering if there was some rationalle behind it :P

Reply:
Ok, I'll use 2 GiB. Setting the limit too large is better than setting the limit too small, I guess.